assets/styles/index.scss

```diff
@@ -213,28 +222,46 @@ body {
   justify-content: space-around;
   top: calc(calc(100%) - 150px);
   
-
+  
+  
   .color-codes {
-    width: calc(calc(100%)/5.3);
+    height: calc(calc(12vw)/5);
+    width: calc(calc(500%)/7);
```

Comment:
Oha oha `%500` ne olum hahaha - adami sikerler lan :D kotu kod yazma, azcik matematik kastir, kolaya kacma
Ek olarak, CSS javascript ogrenen herkese siktiri boktan gelmeye basliyo, fazla umursamiyorlar. Sakin o hataya dusme kardesim. CSS cok ozenle calisilmasi gereken Javascript kadar detayli bir dil. CSS degisiklikleri Error vermedigi icin de en tehlikeli degisiklikler. Mesela, burdaki degisikliklerin mobil gorunumu bozmus ama sanirim farkinda degilsin:
<img width="416" alt="Screen Shot 2020-05-26 at 20 25 30" src="https://user-images.githubusercontent.com/43887066/82966174-30c0ab80-9f98-11ea-96c3-89b6a9bc595c.png">

Reply:
Css keyifli ben arada bakıyorum trick'lerine. Bu da aslında üzerine tıklamak daha rahat olsun diye yaptığım birşeydi. Galiba başka bir ölçüden aldığım için 500% olmuş. dikkat edeceğim.